### ISSUE TITLE
Infinite loop when displaying software keyboard to edit cell on touch device

### ISSUE THREAD

**Issue by seancolsen** [mathesar-foundation/mathesar#1071]
## Steps to reproduce

1. Begin on a touch device with a software keyboard. (On an Android tablet, I can reproduce this with Firefox and Chrome.)
1. View a table.
1. On a cell near the bottom of the table, double-tab to enter edit mode.
1. Observe the device's software keyboard slide up from the bottom of the screen, making the browser viewport shorter, causing the cell to be scrolled out of view.
1. Observe the devices's software keyboard slide away from view, presumably because the input element no longer has focus.
1. Observe the keyboard to continue oscillating between these two states.
1. Tap somewhere else on the screen to return to normal.

This bug makes it impossible to edit a cell near the bottom of the table.

## Status

Blocked by #2703

**Comment by seancolsen** [comment]
@pavish Maybe this has to do with our use of `VirtualList`?

**Comment by pavish** [comment]
@seancolsen I'm not sure, it will require testing.

If it is due to our use of `VirtualList`, the fix would be to scroll into view of focused input when the keyboard opens or to increase the overscanCount.

**Comment by mistryiam** [comment]
Hey @kgodey can I work on this?

**Comment by Anish9901** [comment]
Unassigned due to inactivity.

**Comment by monstajoe2002** [comment]
May I be assigned this issue please?

**Comment by seancolsen** [comment]
Sure, @monstajoe2002. Go ahead.

**Comment by silentninja** [comment]
@seancolsen It is tricky to access the development server on a mobile device. Relevant [Matrix discussion](https://matrix.to/#/!vAyoAQoixqNrvBuIcH:matrix.mathesar.org/$-r0vJt6QWrGUospa3I-7fKrB5GnWbUh3mnh3c2ufs-s?via=matrix.mathesar.org&via=matrix.org). Can you mention it in the description on how to debug this issue?

**Comment by seancolsen** [comment]
@silentninja and @monstajoe2002 I vaguely recall trying to access the dev server through my local network and having trouble with that. I'm pretty sure I resorted to using the _staging_ site from my tablet when reporting this issue. I've opened #2703 and marked this ticket as blocked by that

**Comment by kgodey** [comment]
Closing this, too old, repro steps are likely out of date. We can create a new issue if we see this in the future.
